templates/x86/qemu/mods.config

```diff
@@ -72,7 +72,7 @@ configuration conf {
 	@Runlevel(1) include embox.test.stdlib.bsearch_test
 	@Runlevel(1) include embox.test.stdlib.qsort_test
 	@Runlevel(1) include embox.test.posix.environ_test
-	@Runlevel(1) include embox.test.posix.getopt_test
+	@Runlevel(1) include embox.test.posix.timerfd_test
```

Comment:
For everyone conserned, this getopt was a dup of getopt_test on line 67, so this change look OK